### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by gabrg4** [pypi/support#4117]
### PyPI Username

`gabrg4`

### Reason for request

I have registered a new pypi account but i have misspelled my email address (gabriele.goffredo.06@gmail.com) as gabiele.goffredo.06@gmail.com. I have also created an account on test pypi under the same name, yet again misspelling my email address as gabriele.goffredo@gmail.com.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by Thespi-Brain** [comment]
Hi @gabrg4,

It looks like you were able to successfully create an alternate account with the correct email address, which is great. Are you still having logging issues? Let me know, thanks.

**Comment by gabrg4** [comment]
Yes, as you saw i created another alternative account with the correct primary email address and "_temp" prefix. I am asking you if its possible to change the primary email addresses of the original accounts to the correct one as to recover the username. Even just deleting the account is fine, and i'll make another when i need to use pypi again. Thank you for your help.

**Comment by Thespi-Brain** [comment]
Hi @gabrg4,

I am following up regarding your request. Are you still having an issue regarding your account's email address? Let us know! Regarding your question, we can definitely change the primary email address of your original account if you would like. I will keep this issue open for another week unless I hear from you earlier. After that, if you are still having trouble, feel free to create a new request. Thanks!

**Comment by gabrg4** [comment]
Hello, and thank you for your answer.

Yes, I'm still having issues with the accounts, the issue being that I'd like to delete the original accounts so that I can reuse those usernames with the correct email address.

However, I don't really plan on doing anything more in the near future, so if it's a hassle to you, you can close the ticket. If I were to need the account again, I'm going to make a new ticket.

Thank you for your service.

**Comment by gabrg4** [comment]
Closed on accident.

**Comment by Thespi-Brain** [comment]
Hi @gabrg4,

We don't delete users' accounts for them, since we want our users to have agency over their own account. But as you asked before, it is possible to update the email address for your original account. That way, you can log in and add/delete/use the account as you please. Please contact support@pypi.org with an email address that you would like your account to be associated with and please reference this link (https://github.com/pypi/support/issues/4117). We will proceed from there. 

FYI, we have a one unique email address per one unique account policy. Thanks!

**Comment by Thespi-Brain** [comment]
Hi @gabrg4,

Sure, I will go ahead and close this issue for now then. Feel free to open a new issue when you feel you are ready to use the account again.
